src/main/java/seedu/address/logic/parser/ParserUtil.java

```diff
@@ -127,4 +127,18 @@ public static Index parseIndex(String oneBasedIndex) throws IllegalValueExceptio
         return name.isPresent() ? Optional.of(new DateTime(name.get())) : Optional.empty();
     }
 
+    /**
+     * Parses and trims {@code filePath} if it is not empty.
+     * @throws PropertyNotFoundException if the {@code filePath} is not empty.
+     */
+    public static String parseFilePath(String filePath) throws PropertyNotFoundException {
+        requireNonNull(filePath);
+        filePath = filePath.trim();
+
+        if (filePath.isEmpty()) {
```

Comment:
Should throw `IllegalValueException` rather than `PropertyNotFoundException`

Reply:
It throws it when the file path is not provided, isn't it `PropertyNotFoundException`?

Reply:
`filePath` is not a `Property`, right?

It is a value of type `String`, and the constraint for this value is `It cannot  be blank`.

You can search for the existing usages of `IllegalValueException` and `PropertyNotFoundException` in the codebase. `PropertyNotFoundException` is thrown when this `Property` is not defined (meaning not added in the `PropertyManager`).

Things like `Name`, `Email` are called `Properties`. `filePath` is not a property, right?